uber/decorators.py

```diff
@@ -492,3 +492,21 @@ def __call__(self, func):
         redirect_func = create_redirect(self.url or '../' + get_module_name(func) + '/' + func.__name__)
         setattr(root, self.alias_name or func.__name__, redirect_func)
         return func
+
+
+def attendee_id_required(func):
+    @wraps(func)
+    def check_id(*args, **params):
+        message = "No ID provided. Trying using a different link or going back."
+        session = params['session']
+        if params.get('id'):
+            try:
+                uuid.UUID(params['id'])
+            except ValueError:
+                message = "Your ID is not a valid format. Did you enter or edit it manually?"
+            else:
+                message = "You weren't found in our database."
```

Comment:
Is this error message informative enough?  Should it maybe instead say "The Attendee ID you provided was not found in our database."